components/clarinet-deployments/src/types.rs

```diff
@@ -433,15 +478,148 @@ impl ContractPublishSpecification {
 
 #[derive(Debug, PartialEq, Serialize, Deserialize, Clone)]
```

Comment:
Note: verify in the same method as was done here: https://github.com/hirosystems/clarinet/pull/1116/files#r1297274266

Reply:
Verified. Everything compiled after derives exclusion.